docs/documenters/buildthedocs.rst

```diff
@@ -0,0 +1,20 @@
+Building the Docs Locally
```

Comment:
I think it makes the structure and navigation clearer to have the docs to do with coding and the docs to do with documenting separated. Otherwise the docs about docs are buried at the bottom of the chapter about code, where nobody would expect to look for them. There'll be a couple more files to add to this directory later.